scylla/datadog_checks/scylla/scylla.py

```diff
@@ -0,0 +1,58 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from six.moves.urllib.parse import urlparse
+
+from datadog_checks.base import ConfigurationError, OpenMetricsBaseCheck
+
+from .metrics import ADDITIONAL_METRICS_MAP, INSTANCE_DEFAULT_METRICS
+
+
+class ScyllaCheck(OpenMetricsBaseCheck):
+    """
+    Collect Scylla metrics from Prometheus endpoint
+    """
+
+    DEFAULT_METRIC_LIMIT = 0
+
+    def __init__(self, name, init_config, instances):
+
+        instance = instances[0]
+
+        endpoint = instance.get('instance_endpoint')
```

Comment:
Any reason to not just rely on the name `prometheus_url`?

Reply:
That's a good point - its included in their documentation as a Prometheus API so might be more consistent for the user to configure.